src/riscv-legacy-integration.adoc

```diff
@@ -53,8 +53,9 @@ The CHERI execution mode is key in providing backwards compatibility with the
 base RISC-V ISA. RISC-V software is able to execute unchanged in
```

Comment:
I think that's true, except for the debugger (which I guess is not technically the hart's software) because under the current spec Debug mode would be running in Capability mode even if all CME bits are 0.

Reply:
there are no CME/CRE bits which affect debug mode